### ISSUE TITLE
More user friendly docker image startup

### ISSUE THREAD

**Issue by tdealtry** [hyperk/hk-software#1]
Hi @guiguem As discussed in Tokai, it'd be nice to make the logon to the docker image a bit more user friendly

I am certain that doing this automatically on startup is something everyone (developers & people just running the software) will need
```
. /usr/local/hk/hk-pilot/setup.sh
```

I'd also be somewhat suprised if this is a bad idea (it may mess up some developers, but I don't specifically see how - you can always override paths by redefining environment variables or adding new paths to the start of e.g. `$PATH`)
```
. /usr/local/hk/hk-software/install-Linux_x86_64-gcc_8-python_3.8.13/setup.sh
```
However, if we want to be safe, the alternative would be to `cat` a message on login like
```
HK pilot has been sourced. For usage see:
 hkp --help
The HK software has not been sourced. To do this run:
 . /usr/local/hk/hk-software/install-Linux_x86_64-gcc_8-python_3.8.13/setup.sh
(note that developers may want to customise their setup)
```
I know that this info is in the README now, but it's just much easier to print directly to screen, so people know how (and have something to copy-paste)

**Comment by guiguem** [comment]
Hi!
I guess I don't have a too strong preference; indeed at least the pilot should be sourced to get the proper path to source later with the hk-software.
And then a logon message would be useful indeed! thanks for the suggestion.

**Comment by tdealtry** [comment]
Resolved in d42dfea971f7c1180e6121628efac7aea1533f7f
